external/llvh/lib/Support/Windows/Path.inc

```diff
@@ -1182,6 +1182,7 @@ void closeFile(file_t &F) {
 }
 
 std::error_code remove_directories(const Twine &path, bool IgnoreErrors) {
+#if !defined(WINUWP)
```

Comment:
This looks OK, there is only a small detail. All changes to LLVH should be recorded in the `external/llvh/patches/` directory. Basically:
```
cd external/llvh
git-diff --relative > patches/winup.patch
```